fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/PermissiveAvailability.java

```diff
@@ -68,17 +70,12 @@ public PermissiveAvailability(
     public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constraints) {
         Set<String> allColumnNames = constraints.getAllColumnNames();
 
-        if (allColumnNames.isEmpty()) {
```

Comment:
Refer to `ConcreteAvailability` the behavior of `getAvailableIntervals` behavior, it should filter out to only the columns that is configured in this table. Should add a test to ensure this behavior.